simulation/src/range_likelihood.cpp

```diff
@@ -180,336 +317,384 @@ pcl::simulation::RangeLikelihood::RangeLikelihood (int rows, int cols, int row_h
+  glDeleteRenderbuffers(1, &color_render_buffer_);
+
+  delete[] depth_buffer_;
+  delete[] color_buffer_;
+  delete[] score_buffer_;
 }
 
 double
-pcl::simulation::RangeLikelihood::sampleNormal (double sigma)
+pcl::simulation::RangeLikelihood::sampleNormal(double sigma)
 {
-  std::normal_distribution<> dist (0.0, sigma);
-  return (dist (rng));
+  std::normal_distribution<> dist(0.0, sigma);
+  return (dist(rng));
 }
 
 void
-pcl::simulation::RangeLikelihood::setupProjectionMatrix ()
+pcl::simulation::RangeLikelihood::setupProjectionMatrix()
 {
-  glMatrixMode (GL_PROJECTION);
-  glLoadIdentity ();
+  glMatrixMode(GL_PROJECTION);
+  glLoadIdentity();
 
   // Prepare scaled simulated camera projection matrix
-  float sx = static_cast<float> (camera_width_) / static_cast<float> (col_width_);
-  float sy = static_cast<float> (camera_height_) / static_cast<float> (row_height_);
-  float width = static_cast<float> (col_width_);
-  float height = static_cast<float> (row_height_);
-
-  float fx = camera_fx_/sx;
-  float fy = camera_fy_/sy;
-  float cx = camera_cx_/sx;
-  float cy = camera_cy_/sy;
+  float sx = static_cast<float>(camera_width_) / static_cast<float>(col_width_);
+  float sy = static_cast<float>(camera_height_) / static_cast<float>(row_height_);
+  float width = static_cast<float>(col_width_);
+  float height = static_cast<float>(row_height_);
+
+  float fx = camera_fx_ / sx;
+  float fy = camera_fy_ / sy;
+  float cx = camera_cx_ / sx;
+  float cy = camera_cy_ / sy;
   float m[16];
-  float z_nf = (z_near_-z_far_);
-
-  m[0] = 2.0f*fx/width;  m[4] = 0;              m[ 8] = 1.0f-(2*cx/width);      m[12] = 0;
-  m[1] = 0;              m[5] = 2.0f*fy/height; m[ 9] = 1.0f-(2*cy/height);     m[13] = 0;
-  m[2] = 0;              m[6] = 0;              m[10] = (z_far_+z_near_)/z_nf;  m[14] = 2.0f*z_near_*z_far_/z_nf;
-  m[3] = 0;              m[7] = 0;              m[11] = -1.0f;                  m[15] = 0;
-  glMultMatrixf (m);
+  float z_nf = (z_near_ - z_far_);
+
+  m[0] = 2.0f * fx / width;
+  m[4] = 0;
+  m[8] = 1.0f - (2 * cx / width);
+  m[12] = 0;
+  m[1] = 0;
+  m[5] = 2.0f * fy / height;
+  m[9] = 1.0f - (2 * cy / height);
+  m[13] = 0;
+  m[2] = 0;
+  m[6] = 0;
+  m[10] = (z_far_ + z_near_) / z_nf;
+  m[14] = 2.0f * z_near_ * z_far_ / z_nf;
+  m[3] = 0;
+  m[7] = 0;
+  m[11] = -1.0f;
+  m[15] = 0;
+  glMultMatrixf(m);
 }
 
 void
-pcl::simulation::RangeLikelihood::applyCameraTransform (const Eigen::Isometry3d & pose)
+pcl::simulation::RangeLikelihood::applyCameraTransform(const Eigen::Isometry3d& pose)
 {
   float T[16];
-  Eigen::Matrix4f m = (pose.matrix ().inverse ()).cast<float> ();
-  T[0] = m(0,0); T[4] = m(0,1); T[8] = m(0,2); T[12] = m(0,3);
-  T[1] = m(1,0); T[5] = m(1,1); T[9] = m(1,2); T[13] = m(1,3);
-  T[2] = m(2,0); T[6] = m(2,1); T[10] = m(2,2); T[14] = m(2,3);
-  T[3] = m(3,0); T[7] = m(3,1); T[11] = m(3,2); T[15] = m(3,3);
+  Eigen::Matrix4f m = (pose.matrix().inverse()).cast<float>();
+  T[0] = m(0, 0);
```

Comment:
Same here to conserve the visual pattern. Though a for-loop would be possible too.